docs/testrunner-toolkit/configuration/testcafe.md

```diff
@@ -132,26 +132,135 @@ __Example__:
     platformName: "Windows 10"
 ```
 
-### `testMatch`
+### `screenResolution`
 
-__Description__: The explicit name, regex, or location of the test files.
+<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small><a href="/testrunner-toolkit/running-tests#test-on-sauce-labs">ℹ</a></p>
 
-__Type__: *string* | *regex*
+__Description__: Set browser window screen resolution.
+
+__Type__: *string*
 
 __Example__:
 ```yaml
-    testMatch: '**/*.js'
+    screenResolution: "1920x1080"
 ```
 
-### `screenResolution`
+### `disableScreenshots`
+__Description__: Prevents TestCafe from taking screenshots. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#disablescreenshots).
```

Comment:
Is there any value in calling out that the default setting is false?

Reply:
No value is set by default actually according to testcafe documentation. If it's not set, it should use default value for boolean. In most cases, it's `false`.